### DISCUSSION TITLE
Can I safelist using regex?

### DISCUSSION THREAD

**Discussion by YogliB** [windicss/windicss#329] [Q&A] [answered]
Is doing something like this possible?

```javascript
safelist: [/^h-.*/, /^w-.*/]
```

**Comment by antfu** [comment, marked as answer]
No, windicss is on-demand and there are infinite possibilities to match. As a workaround, see https://windicss.org/integrations/vite.html#safelist

**Reply by YogliB** [reply]
Was looking for a shortcut... Thanks for the quick answer!

**Comment by terranc** [comment]
Hope to support regular support like tailwindcss.
https://tailwindcss.com/docs/content-configuration#using-regular-expressions
